src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -89,9 +105,7 @@ export const NewCommentForm: React.FC<Props> = ({
         className={`
         NewCommentForm__submit-button
         button
-        ${nameValue === ''
-        || emailValue === ''
-        || commentValue === '' ? 'disabled' : ''}
+        ${!canSubmit ? 'disabled' : ''}
         `}
```

Comment:
use `classnames` lib - it could make your life easier 
https://www.npmjs.com/package/classnames

Reply:
done, ty